### PR TITLE
bc: Depends on bison and flex for Linuxbrew

### PR DESCRIPTION
**Pull request by sjackman**
Fix configure: error: cannot find output from lex; giving up

### CODE REVIEW

File: bc.rb [hunk 1]
```diff
@@ -14,6 +14,9 @@ class Bc < Formula
 
   keg_only :provided_by_osx
 
+  depends_on "bison" => :build unless OS.mac?
```
Comment (DomT4) [inline bc.rb]: These should be in a block 😉 

```ruby
unless OS.mac?
  depends_on "bison" => :build
  depends_on "flex" => :build
end
```

Comment (sjackman) [inline bc.rb]: Right. I think you've given me that note before.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-dupes/commit/c3c0e22e96d463cc6fb62dbcd457effc93b2f011. Thanks for making that change.

**Comment by sjackman** [comment]
Thanks for merging.

**Pull request merged**